sherpa/bin/streaming_conformer_rnnt/decode.py

```diff
@@ -15,8 +15,9 @@
 # limitations under the License.
 
 import math
-from typing import List
+from typing import List, Optional
```

Comment:
Please also update CI to test the newly added `fast_beam_search`.

You can use

https://github.com/k2-fsa/sherpa/pull/66/files#diff-0fc8407285d79be4141309a772e34bdb812ee4b6444e80f69191cbe0ad6d51a9

as an example to update the CI.